context/AuthHelper.tsx

```diff
@@ -14,6 +14,12 @@ type UserCredentials = {
   password: string;
 };
 
+interface logoutType {
```

Comment:
Move interfaces and types to their own external file.

Reply:
fixed